packages/world/package.json

```diff
@@ -11,6 +11,7 @@
   "type": "module",
   "exports": {
     ".": "./dist/ts/index.js",
+    "./actions": "./dist/ts/actions/index.js",
```

Comment:
Following the common library's actions path, I've made a new path for this. But we only have this function now, so I can move the function into `./ts/delegationActions.ts` if preferred.